_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,153 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul` , `ol` , and `dl` HTML elements where the [semantic role][] is the same as the [implicit semantic role][].
+
+## Expectation
+
+Each target element:
+
+- is categorized to follow the [flow content model][], and;
+- that it's contents are its [descendants][] in the DOM, and;
```

Comment:
I usually understand "content" as "web content" (https://www.w3.org/TR/WCAG21/#dfn-content) but here it seems to mean "list content" or something like that.

https://html.spec.whatwg.org/multipage/dom.html#concept-html-contents define content as children in the DOM. We probably should stick to children when talking about the DOM to avoid confusion with "web content".

But then, if content=children, the item makes little sense.